spec/workers/detect_invariants_spec.rb

```diff
@@ -233,5 +233,27 @@
         ),
       )
     end
+
+    it 'detects when a submitted application does not have 2 selected references' do
+      three_selected_references_application_form = create(:completed_application_form, :with_completed_references)
+      create(:submitted_application_choice, application_form: three_selected_references_application_form)
+      create(:reference, :feedback_provided, selected: true, application_form: three_selected_references_application_form)
+      create(:reference, :feedback_provided, selected: true, application_form: three_selected_references_application_form)
+      create(:reference, :feedback_provided, selected: true, application_form: three_selected_references_application_form)
+
+      create(:completed_application_form, :with_completed_references)
```

Comment:
Worth adding a couple of references to this, so it's clear that forms with the correct number of selected references aren't picked up in the check.